src/main/java/jenkins/plugins/office365connector/model/PotentialAction.java

```diff
@@ -32,10 +33,14 @@
 
     private List<String> target;
 
-    public PotentialAction(List<String> url) {
+    public PotentialAction(String name, String url) {
+        this(name, Collections.singletonList(url));
+    }
+
+    public PotentialAction(String name, List<String> url) {
```

Comment:
can url be null so we can fix it on the fly like https://github.com/jenkinsci/office-365-connector-plugin/pull/24/files#diff-4d256ee5f164b70f36b78f29a615087cR166 ?

Reply:
Sounds like a plan 👍